### ISSUE TITLE
GPT2 output logits?

### ISSUE THREAD

**Issue by cdj0311** [NVIDIA/FasterTransformer#306]
hi, 
The GptOp.forward just support beam search decode, how to output the logits (shape=[batch_size, vocab_size])?

**Comment by byshiue** [comment]
GptOp only supports generating multiple tokens once. It cannot generate logits.

**Comment by cdj0311** [comment]
> 

Thanks for your reply, but how can I get the logits with other way?

**Comment by byshiue** [comment]
You need to modify the source codes and pytorch custom op.
